package.json

```diff
@@ -21,10 +21,10 @@
   },
   "homepage": "https://github.com/blogfoster/eslint-config-blogfoster#readme",
   "dependencies": {
-    "babel-eslint": "^6.1.2",
-    "eslint-config-airbnb": "^12.0.0",
-    "eslint-plugin-import": "^1.16.0",
-    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "babel-eslint": "^7.1.0",
+    "eslint-config-airbnb": "^13.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.3.0",
```

Comment:
I just used react-check and that's what it gave me? should i update for the latest?